website/blog/2022-01-06-open-sourcing-sorbet-vscode.md

```diff
@@ -0,0 +1,118 @@
+---
+id: open-sourcing-sorbet-vscode
+title: Open-sourcing the Sorbet VS Code Extension
+---
+
+We’re excited to announce that Stripe’s VS Code extension for Sorbet is now
+[open source](http://#TODO). We’ve designed Sorbet to be used in editors from
+Day 1—Sorbet exposes a flag (`--lsp`) that starts Sorbet in
+[Language Server Protocol](https://microsoft.github.io/language-server-protocol/)
+(LSP) mode. In this mode, Sorbet can respond to many LSP requests, like Go To
+Definition, Find All References, Autocomplete, and more.
+
+With this release, we’re making it even easier to take advantage of these LSP
+features when working with Sorbet in VS Code. We hope that this extension takes
+some of the guess work out of how to configure and use Sorbet’s LSP mode.
+
+<!--truncate-->
+
+Today’s release includes:
+
+- The [pre-built extension](http://#TODO) on the Visual Studio Marketplace
```

Comment:
there's a TODO link

do we know what URL this will eventually be based on what ID we'll release the extension under?

Reply:
we should be able to: `https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync` is an example of a page

Reply:
Fixed.